Modules/IO/ImageBase/test/CMakeLists.txt

```diff
@@ -40,6 +40,35 @@ set(ITKIOImageBaseTests
     itkImageFileReaderManyComponentVectorTest.cxx)
 
 createtestdriver(ITKIOImageBase "${ITKIOImageBase-Test_LIBRARIES}" "${ITKIOImageBaseTests}")
+
+if("${ITK_COMPUTER_MEMORY_SIZE}" GREATER 5)
+  itk_add_test(
+    NAME
+    itkLargeImageWriteConvertReadTest
+    COMMAND
+    ITKIOImageBaseTestDriver
+    itkLargeImageWriteConvertReadTest
+    ${ITK_TEST_OUTPUT_DIR}/itkLargeImageWriteConvertReadTest.mha
+    30000L)
+  itk_add_test(
+    NAME
+    itkLargeImageWriteReadTest_2D
+    COMMAND
+    ITKIOImageBaseTestDriver
+    itkLargeImageWriteReadTest
+    ${ITK_TEST_OUTPUT_DIR}/itkLargeImageWriteReadTest_2D.mha
+    30000L)
+  itk_add_test(
+    NAME
+    itkLargeImageWriteReadTest_3D
+    COMMAND
+    ITKIOImageBaseTestDriver
+    itkLargeImageWriteReadTest
+    ${ITK_TEST_OUTPUT_DIR}/itkLargeImageWriteReadTest_3D.mha
+    30000L
+    4L)
```

Comment:
It looks like each of these tests needs 5 GB of memory and can not be run concurrently on memory constrained system. Please add something like the following for each memory intensive test:
```
   set_tests_properties(itkComposeBigVectorImageFilterTest
            PROPERTIES
            RESOURCE_LOCK
            MEMORY_SIZE)
```
This will ensure only one test with "MEMORY_SIZE" resource constrained is run at a time.